docs/pages/reference/cli.mdx

```diff
@@ -327,6 +327,106 @@ $ tsh play --format=pty 1fe153d1-ce8b-4ef4-9908-6539457ba4ad.tar
 $ tsh play --format=json ~/play/0c0b81ed-91a9-4a2a-8d7c-7495891a6ca0.tar | jq '.event
 ```
 
+### tsh proxy db
+
+Start local TLS proxy for database connections when using Teleport in single-port mode.
```

Comment:
```
$ grep -Rn "single-port mode" docs/pages
docs/pages/reference/cli.mdx:332:Start local TLS proxy for database connections when using Teleport in single-port mode.
docs/pages/reference/cli.mdx:379:Start local TLS proxy for `ssh` connections when using Teleport in single-port mode.
```
Since "single-port mode" is only used on this page, is there another term we can use to be more consistent with other parts of the docs?

Reply:
_TLS Routing_ is the right term to use here IMO